src/client.cpp

```diff
@@ -59,8 +61,8 @@ void waybar::Client::handleOutput(struct waybar_output &output) {
       .logical_position = [](void *, struct zxdg_output_v1 *, int32_t, int32_t) {},
       .logical_size = [](void *, struct zxdg_output_v1 *, int32_t, int32_t) {},
       .done = [](void *, struct zxdg_output_v1 *) {},
-      .name = &handleOutputName,
-      .description = [](void *, struct zxdg_output_v1 *, const char *) {},
+      .name = [](void *, struct zxdg_output_v1 *, const char *) {},
+      .description = &handleOutputDescription,
```

Comment:
I wanted to keep the old name callback as well. Unfortunately there _seems_ to be a conflict in how Waybar handles them. I don't know the details and didn't want to fall in a rabbit hole.

I have 2 Dell U2515H monitors and it would only callback the description once instead of twice. So DP-4 and DP-5 would be reported twice via the name callback, but their description (and thus identifiers) would only be reported for 1 instead of 2. Only relying on the description solves it.